atomsci/ddm/utils/datastore_functions.py

```diff
@@ -2087,13 +2094,13 @@ def copy_datasets_to_bucket(dataset_keys, from_bucket, to_bucket, ds_client=None
     if ds_client is None:
         ds_client = config_client()
 
-    if type(dataset_keys) == str:
+    if isinstance(dataset_keys, str):
```

Comment:
test?

Reply:
This will need to wait for a later release when the datastore is back up.